onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -41,7 +41,7 @@ def testTvmImported(self):
     def testModelSerialization(self):
         try:
             so = onnxrt.SessionOptions()
-            so.log_verbosity_level = 1
+            so.log_severity_level = 1
             so.logid = "TestModelSerialization"
```

Comment:
misc fix to a misleading example usage. 

verbosity level is only applicable when severity is set to VERBOSE. severity is the typical thing to set.